tactic/interactive.lean

```diff
@@ -6,6 +6,7 @@ Authors: Mario Carneiro, Simon Hudon, Sebastien Gouezel, Scott Morrison
 import data.dlist data.dlist.basic data.prod category.basic
   tactic.basic tactic.rcases tactic.generalize_proofs
   tactic.split_ifs logic.basic tactic.ext tactic.tauto tactic.replacer
+  tactic.where
```

Comment:
I don't think this should be imported with the default set. It's a transitory command, so it won't be needed in most of mathlib. Perhaps we should have an import set for transitory commands like `#find` and `#where` but they shouldn't be imported in basic files that don't need them.

Reply:
Sure, I was totally just acting on Reid's advice